spec.html

```diff
@@ -7243,6 +7242,289 @@ <h1>
   <h1>Syntax-Directed Operations</h1>
   <p>In addition to those defined in this section, specialized syntax-directed operations are defined throughout this specification.</p>
 
+  <emu-clause id="sec-evaluation" type="sdo">
+    <h1>Runtime Semantics: Evaluation ( ): a Completion Record</h1>
+    <dl class="header">
+    </dl>
+    <emu-note>
+      The definitions for this operation are distributed over the "ECMAScript Language" sections of this specification. Each definition appears after the defining occurrence of the relevant productions.
+    </emu-note>
+    <ul>
+      <li>
+        <emu-xref href="#sec-identifiers-runtime-semantics-evaluation"></emu-xref>
+        IdentifierReference
```

Comment:
I think these will link if you surround each nonterminal with `|`.

Reply:
Done. Mind you, clicking the nonterminal (rather than the section number) will always land you farther from the Eval semantics in question.